packages/story-editor/src/utils/getResourceBaseColor.js

```diff
@@ -46,16 +46,19 @@ function getImgNodeKey(elementId) {
   return `${IMG_NODE}_${elementId}`;
 }
 
-export function getMediaBaseColor(resource, onBaseColor) {
+export function getResourceBaseColor(resource) {
   const { type, src, poster } = resource;
-  setOrCreateImage({
-    src: type === 'video' ? poster : src,
+
+  const imageSrc = type === 'image' ? src : poster;
+
+  if (!imageSrc) {
+    return Promise.reject(new Error('No source to image'));
+  }
+  return setOrCreateImage({
+    src: imageSrc,
     width: 10,
     height: 'auto',
-  }).then(
-    (color) => onBaseColor(color),
-    () => onBaseColor([255, 255, 255]) // Fallback color is white.
```

Comment:
FI: the Color Thief throws an error when the image is fully white (it's a feature), so in case of a white image we'd need to set it inside a catch.

Reply:
Thanks for this, I have fixed in PR

Reply:
@spacedmonkey Where is this fixed?

I still see no `catch` block in this function. This should still be there.

Reply:
Catch it upstream.

Reply:
Try / catch has now been added.